docs/rum_collection.md

```diff
@@ -0,0 +1,261 @@
+
+* get a global idea about your app’s performance and demographics;
+* understand which resources are the slowest;
+* analyze errors by OS and device type.
+
+## Setup
+
+
+1. Declare the library as a dependency depending on your package manager (check our [Releases page][releases] for the recent beta version):
+
+    {{< tabs >}}
+    {{% tab "CocoaPods" %}}
+
+You can use [CocoaPods][cocoapods] to install `dd-sdk-ios`:
+```
+pod 'DatadogSDK', :git => 'https://github.com/DataDog/dd-sdk-ios.git', :tag => '1.4.0-beta1'
+```
+
+[cocoapods]: https://cocoapods.org/
+
+    {{% /tab %}}
+    {{% tab "Swift Package Manager (SPM)" %}}
+
+To integrate using Apple's [Swift Package Manager][spm], add the following as a dependency to your `Package.swift`:
+```swift
+.package(url: "https://github.com/DataDog/dd-sdk-ios.git", .exact("1.4.0-beta1"))
+```
+
+[spm]: https://swift.org/package-manager/
+
+    {{% /tab %}}
+    {{% tab "Carthage" %}}
+
+You can use [Carthage][carthage] to install `dd-sdk-ios`:
+```
+github "DataDog/dd-sdk-ios" "1.4.0-beta1"
+```
+
+[carthage]: https://github.com/Carthage/Carthage
+
+    {{% /tab %}}
+    {{< /tabs >}}
+
+2. Initialize the library with your application context and your [Datadog client token][client-token]. For security reasons, you must use a client token: you cannot use [Datadog API keys][api-keys] to configure the `dd-sdk-ios` library as they would be exposed client-side in the iOS application IPA byte code. For more information about setting up a client token, see the [client token documentation][client-token]. You also need to provide an Application ID (see our [RUM Getting Started page][rum-getting-started]).
+
+    {{< tabs >}}
+    {{% tab "US" %}}
+
+```swift
+Datadog.initialize(
+    appContext: .init(),
+    configuration: Datadog.Configuration
+        .builderUsing(
+            rumApplicationID: "<rum_application-id>",
+            clientToken: "<client_token>",
+            environment: "<environment_name>"
+        )
+        .set(serviceName: "app-name")
+        .build()
+)
+```
+
+    {{% /tab %}}
+    {{% tab "EU" %}}
+
+```swift
+Datadog.initialize(
+    appContext: .init(),
+    configuration: Datadog.Configuration
+        .builderUsing(
+            rumApplicationID: "<rum_application-id>",
+            clientToken: "<client_token>",
+            environment: "<environment_name>"
+        )
+        .set(serviceName: "app-name")
+        .set(rumEndpoint: .eu)
+        .build()
+)
+```
+
+    {{% /tab %}}
+    {{< /tabs >}}
+
+3. Configure and register the RUM Monitor. You only need to do it once, usually in your `AppDelegate` code:
+
+    ```swift
+    import Datadog
+
+    Global.rum = RUMMonitor.initialize()
+    ```
+
+From here, you have two choices. You can either use our auto instrumentation to let the SDK track RUM Views, Resources, Actions and Errors automatically or you can instrument your app manually to send this data. It is also possible to mix both methods.
```

Comment:
Just a matter of preference but I'd rather have:

The RUM SDK offers two instrumentation methods:
1. Auto-instrumentation (recommended) - the SDK tracks views, resources, actions and errors automatically
2. Manual instrumentation - you instrument your code to send RUM events

**Note**: It is possible to mix both methods.

what do you think?

Reply:
Yes, it's a better narration, done 👌